cms/templates/js/course-video-settings-update-org-credentials-footer.underscore

```diff
@@ -1,4 +1,8 @@
+<button class='button-link action-cancel-course-video-settings' aria-describedby='cancel-button-text'>
+    <%- gettext('Reset') %>
+    <span id='updcancel-button-text' class='sr'><%-gettext('Click Reset to discard your changes.') %></span>
```

Comment:
@mushtaqak From a11y perspective, it's better to not use click because user might not have mouse to click. I'd prefer to use Press instead of click.